packages/rpc/node/src/ts/RpcServer.ts

```diff
@@ -889,6 +939,25 @@ export class RpcServer<Context extends HttpContext> {
         return getOk();
     }
 
+    private async runServerFinalizers(
+        args: any,
+        context: Context,
+        serviceReply: Reply<unknown>,
+    ): Promise<Reply> {
+        for (const finalizer of this.config.finalizerQueue) {
+            let result;
+            try {
+                result = await finalizer.execute(args, context, serviceReply);
+            } catch (e) {
```

Comment:
Nit: Please try to avoid one letter shorthands.
```suggestion
            } catch (error) {
```